### DISCUSSION TITLE
GitHub Student Developer Pack: Your application cannot be reviewed

### DISCUSSION THREAD

**Discussion by JiaJun98** [community/community#42311] [GitHub Education]
Hello, 
I was trying to get the GitHub Student Developer Pack. I am currently a student until June 2023 and I got this following error.

Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name and a short bio.,Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?
Your image should be one of the following:
Dated school ID

What do they mean by "Dated School ID"? I have uploaded many of my school official doucuments but they all had the same error.

Thank you and I look forward to your help

**Comment by SilasRodrigues19** [comment]
Hi @JiaJun98 

There is a discussion on this topic at this link: https://github.com/orgs/community/discussions/30138

Regarding the Dated School ID, in my case I had to take a picture instead of uploading it

**Reply by JiaJun98** [reply]
@SilasRodrigues19 Thank you I will try it again
